package.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
now we're using es6 can you add an engines so we guard against older node version :love_hotel: